{"id":2164,"details":"\n  \n  \u003ctable\u003e\n    \u003ctr\u003e\n      \u003ctd\u003eRegion\u003c/td\u003e\n\u003ctd\u003eCorse\u003c/td\u003e\n    \u003c/tr\u003e\n    \u003ctr\u003e\n\u003ctd\u003eDepartment\u003c/td\u003e\n\u003ctd\u003eCorse-du-Sud\u003c/td\u003e\n\u003c/tr\u003e\n    \u003ctr\u003e\n\u003ctd\u003eCity\u003c/td\u003e\n\u003ctd\u003eAjaccio\u003c/td\u003e\n\u003c/tr\u003e\n    \n  \u003c/table\u003e\n  ","patient":"Patient From Corse, Corse Du Sud, Ajaccio","source":"unknown_source","status":"unreported","confirmed_at":"2020-03-06T00:00:00.000Z","created_at":"2020-03-16T10:13:30.412Z","updated_at":"2020-03-16T10:42:55.729Z","infected_by":"unknown_infection","number":1890,"urls":["https://github.com/lperez31/coronavirus-france-dataset"],"recovered_at":null,"gender":"unidentified_gender","age":0,"nationality":"unidentified_nationality","symptomatic_at":null,"slug":"france-case-1890-patient-from-corse-corse-du-sud-ajaccio","suggested_url":"france-case-1890-patient-from-corse-corse-du-sud-ajaccio","hospital_stay_length_in_days":null,"days_from_symptoms_to_confirmation":null,"without_symptoms":false,"reporting_country":"fr"}